### PR TITLE
added fiber links and acl

### PR DESCRIPTION
**Pull request by PanStick**

### CODE REVIEW

File: BGP.txt [hunk 1]
```diff
@@ -58,6 +59,8 @@ router eigrp 100
 network 5.0.11.0
 network 5.0.7.0
 network 5.0.12.0
+network 5.0.16.0
+network 5.0.17.0
 exit
```
Comment (watislaf) [inline BGP.txt:64]: co to sa za sieci  16 i 17?

File: ACL.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Router 1_2
```
Comment (watislaf) [inline ACL.txt:1]: Zmodyfikowalbys README moze tez ?

Comment (PanStick) [reply ACL.txt:1]: done

File: README.md [hunk 3]
```diff
@@ -1,21 +1,25 @@
-
 # ciscoLab ![](https://geps.dev/progress/94) (63/70)
```
Comment (watislaf) [inline README.md]: 100 na 100

File: BGP.txt (hunk 1 above)
Comment (PanStick) [reply BGP.txt:64]: image

Comment (PanStick) [reply BGP.txt:64]: dodane z fiber linkami

Comment (watislaf) [reply BGP.txt:64]: kk

**Pull request merged**